packages/blockchain-extension/test/commands/openTransactionViewCommand.test.ts

```diff
@@ -153,6 +153,17 @@ describe('OpenTransactionViewCommand', () => {
             openViewStub.should.have.been.calledOnce;
         });
 
+        it('should correctly display the Local Fabric gateway name', async () => {
```

Comment:
Can use `FabricRuntimeUtil.LOCAL_FABRIC` and `FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME` in the tests (and test name) as well.